scripts/aem.js

```diff
@@ -424,6 +424,27 @@ function decorateButtons(element) {
   });
 }
 
+/**
+ * Decorate Social Media icons with links
```

Comment:
Just a nitpick, but can you change to "Decorate pictures with links"?
because theoretically the image can be some content on any page.